tycho-extras/tycho-pomless/pom.xml

```diff
@@ -47,7 +47,7 @@
     <dependency>
       <groupId>io.takari.polyglot</groupId>
       <artifactId>polyglot-common</artifactId>
-      <version>0.4.8</version>
+      <version>0.4.9-SNAPSHOT</version>
```

Comment:
I don'T think we can use snapshot here, please ask for a release at polyglot

Reply:
Yes. We have to wait for the mentioned PR to be merged, then I will ask for a release.
I just wanted to indicate that eventually (when this PR is not a draft anymore) this is required.